editor/editor_node.cpp

```diff
@@ -2102,8 +2114,7 @@ void EditorNode::edit_item(Object *p_object, Object *p_editing_owner) {
 			if (!item_plugins.has(plugin)) {
 				// Remove plugins no longer used by this editing owner.
 				to_remove.push_back(plugin);
-				plugin->make_visible(false);
-				plugin->edit(nullptr);
+				_plugin_over_edit(plugin, nullptr);
```

Comment:
I assume it is desirable to remove the plugin here (a new side-effect of using the common method)?

Reply:
Yes, what I mean, previous version of this bit of code didn't do it. This version does, because `_plugin_over_edit()` does that unconditionally. So I'm just checking that this is a desirable change.

Reply:
Yes, this was the cause of the first linked issue.